ui/package.json

```diff
@@ -11,11 +11,10 @@
     "test-react": "jest",
     "lint": "ng lint --type-check",
     "e2e": "ng e2e",
-    "codegen": "yarn run merge-swagger && yarn run codegen-clean && yarn run codegen-swagger && yarn run codegen-swagger-fetch && yarn run post-codegen && yarn run notebooks-codegen-swagger && yarn run notebooks-codegen-swagger-fetch && yarn run notebooks-post-codegen",
-    "merge-swagger": "../api/build.rb merge-yaml --no-docker",
+    "codegen": "yarn run codegen-clean && yarn run codegen-swagger && yarn run codegen-swagger-fetch && yarn run post-codegen && yarn run notebooks-codegen-swagger && yarn run notebooks-codegen-swagger-fetch && yarn run notebooks-post-codegen",
     "codegen-clean": "rm -rf src/generated src/notebooks-generated",
-    "codegen-swagger": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-angular --input-spec ../api/src/main/resources/merged.yaml --output src/generated --additional-properties ngVersion=2",
-    "codegen-swagger-fetch": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-fetch --input-spec ../api/src/main/resources/merged.yaml --output src/generated/fetch",
+    "codegen-swagger": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-angular --input-spec ../api/src/main/resources/workbench-api.yaml --output src/generated --additional-properties ngVersion=2",
```

Comment:
Why not use gradle to wrap these for both the front and back ends? It seems like we're duplicating efforts. I'm irked (though not miffed) that I needed to change this file name in more than one place.